config.toml

```diff
@@ -0,0 +1,342 @@
+[outputFormats.HEADERS]
+mediatype = "text/netlify"
+baseName = "_headers"
+isPlainText = true
+notAlternative = true
+
+# Image processing configuration.
+[imaging]
+resampleFilter = "CatmullRom"
+quality = 75
+anchor = "smart"
+
+[services]
+[services.googleAnalytics]
+# Comment out the next line to disable GA tracking. Also disables the feature described in [params.ui.feedback].
+id = "UA-00000000-0"
+
+# Additional menu items
+
+[[menu.main]]
+    name = "Documentation"
+    weight = 40
+    url = "/docs/"
+[[menu.main]]
+    name = "Blog"
+    weight = 50
+    url = "/blog/"
+[[menu.main]]
+    name = "Community"
+    weight = 60
+    url = "/community/"
+[[menu.main]]
+    name = "Partners"
+    weight = 70
+    url = "/partners/"
+
+
+# Specify a value here if your content directory is not in your repo's root directory
+#github_subdir = "userguide"
+
+time_format_blog = "Monday, January 02, 2006"
+time_format_default = "January 2, 2006"
+# Sections to publish in the main RSS feed.
+rss_sections = ["blog"]
+
+
+# Language configuration
+
+
+# Everything below this are Site Params
+
+[params]
+copyright = "The Falco Authors"
+copyright_linux =  "The Linux Foundation ®."
+
+# Menu title if your navbar has a versions selector to access old versions of your site.
+# This menu appears only if you have at least one [params.versions] set.
+version_menu = "Releases"
+
+# Flag used in the "version-banner" partial to decide whether to display a
+# banner on every page indicating that this is an archived version of the docs.
+# Set this flag to "true" if you want to display the banner.
+archived_version = false
+
+# The version number for the version of the docs represented in this doc set.
+# Used in the "version-banner" partial to display a version number for the
+# current doc set.
+version = "v0.27.0"
+
+# A link to latest version of the docs. Used in the "version-banner" partial to
+# point people to the main doc site.
+url_latest_version = "https://falco.org"
+
+# Repository links (https://www.docsy.dev/docs/adding-content/repository-links)
+# Repository configuration (URLs for in-page links to opening issues and suggesting changes)
+github_repo = "https://github.com/falcosecurity/falco-website"
+# Specify a value here if your content directory is not in your repo's root directory
+# github_subdir = "userguide"
+# An optional link to a related project repo. For example, the sibling repository where your product code lives.
+github_project_repo = "https://github.com/falcosecurity/falco-website"
+# Specify a value here if you have would like to reference a different branch for the other github settings
+# github_branch =
+
+time_format_blog = "Monday, January 02, 2006"
+time_format_default = "January 2, 2006"
+# Sections to publish in the main RSS feed.
+rss_sections = ["blog"]
+
+# Google Custom Search Engine ID. Remove or comment out to disable search.
+gcs_engine_id = "011217106833237091527:la2vtv2emlw"
```

Comment:
I have noticed that ID :point_up: comes from :point_right: https://github.com/google/docsy/blob/976ee59c947a153a5c4fe9adc620b32684505d73/userguide/config.toml#L129

@Rajakavitha1 I guess we need our own gcs_engine_id?

Or do we want to switch to offline search with Lunr.js?

cc @leodido @fntlnz

Reply:
Yeah, that needs to be changed IMHO.

Reply:
@leogr Yes, that is right, we need to change that. Using Lunr.js is a good option too. However, Google Engine might give us good SEO search options. @leodido  and @fntlnz Please share your thoughts.

Reply:
I think that Lunr is good however Google results seems more accurate as far as I can see on the Kubernetes website.